version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1.1-SNAPSHOT"
```

Comment:
Why have the version in a separate file? Also the version should be 0.1.0, as it is in build.sbt.

Reply:
This is part of the `sbt release` process.